wdl2cwl/main.py

```diff
@@ -358,13 +358,35 @@ def convert(workflow: str) -> str:
 
     if "time_minutes" in ast.task_runtime:
 
-        time_minutes = ""
+        time_minutes: Union[str, int] = ""
         if '"' not in ast.task_runtime["time_minutes"]:
-            time_minutes = "$(inputs." + ast.task_runtime["time_minutes"] + "* 60)"
+            if ast.task_runtime["time_minutes"].replace('"', "") in input_names:
```

Comment:
Is this `replace` necessary? We are already inside the `if '"' not in ast.task_runtime["time_minutes"]` block